### ISSUE TITLE
Create a "Deploying with Docker" chapter

### ISSUE THREAD

**Issue by m-renaud** [yesodweb/yesodweb.com-content#168]
The existing "Deploying your Webapp" chapter has great information on deploying an app with full stack setup. Increasingly, docker is being used to deploy apps so there should probably be a chapter on it. As a nice to have there could be a section on how to deploy to some of the major cloud providers like AWS and Google Cloud. Also, since the currently deploying chapter already has a lot of information it may make sense to have it as a separate chapter under the advanced section. Thoughts?

**Comment by snoyberg** [comment]
Sounds like a great idea, or possibly as a cookbook or wiki entry instead
(with a link from the book).

**Comment by m-renaud** [comment]
Yeah, wiki entry sounds good, there's already a Deployment section in there as well. If I have some time in the coming weeks I'll put one together.
